Resources/Prototypes/Research/arsenal.yml

```diff
@@ -107,7 +107,7 @@
     state: icon
   discipline: Arsenal
   tier: 1
-  cost: 8000
```

Comment:
DeltaV comment.

Reply:
> DeltaV comment.

Just fyi the markers.md link is broken, so I don't have a reference for what needs comments and what doesn't (I think thats where it specifies). comment added at the bottom of the tech since putting it after the price breaks it for some reason.